tensorflow_serving/tools/docker/Dockerfile.mkl

```diff
@@ -0,0 +1,58 @@
+ARG TF_SERVING_VERSION=latest
+ARG TF_SERVING_BUILD_IMAGE=tensorflow/serving:${TF_SERVING_VERSION}-devel-mkl
+
+FROM ${TF_SERVING_BUILD_IMAGE} as build_image
+FROM ubuntu:16.04
+
+ARG TF_SERVING_VERSION_GIT_BRANCH=master
+ARG TF_SERVING_VERSION_GIT_COMMIT=head
+
+LABEL maintainer="gvasudevan@google.com"
+LABEL tensorflow_serving_github_branchtag=${TF_SERVING_VERSION_GIT_BRANCH}
+LABEL tensorflow_serving_github_commit=${TF_SERVING_VERSION_GIT_COMMIT}
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install TF Serving pkg
+COPY --from=build_image /usr/local/bin/tensorflow_model_server /usr/bin/tensorflow_model_server
+
+# MKL installation
+COPY --from=build_image /mkl-dnn/external/mklml_lnx_*/lib/libiomp5.so /usr/local/lib
+COPY --from=build_image /mkl-dnn/external/mklml_lnx_*/lib/libmklml_gnu.so /usr/local/lib
+COPY --from=build_image /mkl-dnn/external/mklml_lnx_*/lib/libmklml_intel.so /usr/local/lib
+
+ENV LIBRARY_PATH '/usr/local/lib:$LIBRARY_PATH'
+ENV LD_LIBRARY_PATH '/usr/local/lib:$LD_LIBRARY_PATH'
+
+
+# Expose ports
+# gRPC
+EXPOSE 8500
+
+# REST
+EXPOSE 8501
+
+# Set where models should be stored in the container
+ENV MODEL_BASE_PATH=/models
+RUN mkdir -p ${MODEL_BASE_PATH}
+
+# The only required piece is the model name in order to differentiate endpoints
+ENV MODEL_NAME=model
+
+# Set inter_op or intra_op threads
+ENV TENSORFLOW_SESSION_PARALLELISM=2
+
+# Create a script that runs the model server so we can use environment variables
+# while also passing in arguments from the docker command line
+RUN echo '#!/bin/bash \n\n\
+tensorflow_model_server --port=8500 --rest_api_port=8501 \
```

Comment:
Maybe you should add ENV GPRC_PORT 8500, then use it here as --port=${GRPC_PORT}?

Reply:
Why not just bind ports in docker to remap them?

Reply:
yeah, not sure this needs to be changed, but if there's a good reason it should be reflected in all serving dockerfiles.

Reply:
Can you explain more about this `bind ports in docker to remap them`? I mean what are you expecting here.

Reply:
I'm expecting no change, this was a response to @dzungductran's suggestion.